package.json

```diff
@@ -41,7 +41,8 @@
     "fft-windowing-ts": "0.2.7",
     "lit": "3.2.1",
     "music-metadata": "11.0.0",
-    "prismjs": "1.29.0"
+    "prismjs": "1.29.0",
+    "standardized-audio-context": "^25.3.77"
```

Comment:
have we documented why we need this package?

Reply:
Good catch, this is not actually needed and never used. I used this to test why Safari wasn't playing audio.

I'll remove the package shortly